app/components/Footer/index.js

```diff
@@ -22,6 +22,7 @@ function Footer() {
             author: <A href="https://twitter.com/mxstbr">Max Stoiber</A>,
           }}
         />
+        <a onClick={() => {}}>dude</a>
```

Comment:
did not catch this issue that I pushed after opening the PR, may be a setting that needs to be configured